### DISCUSSION TITLE
レビューページのナンバリングの呼び方。

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1853] [Q&A] [answered]
ページ番号を呼び出して、すべてのページの中心にしようとしています。

re:view で呼び出す方法は? \review@thepage を試していますが、エラーが発生しました。
ファイル名 review-custom.sty

レビューページのナンバリングの呼び方。

![image](https://user-images.githubusercontent.com/49851974/186302700-83efa690-7b34-4828-ba3a-8ceb37e4bd7e.png)
![image](https://user-images.githubusercontent.com/49851974/186301958-13cef4f3-4810-4e29-9e1f-af7079aa8418.png)
Preface
![image](https://user-images.githubusercontent.com/49851974/186302177-898b92da-87b2-4f31-a55e-be04cad17874.png)
Chapters
![image](https://user-images.githubusercontent.com/49851974/186302275-aa3e9de8-a5fe-44e5-a893-0e9a4b20e001.png)

**Comment by kmuto** [comment, marked as answer]
```
\fancyfoot[C]{\thepage}
```
ですかね。fancyhdrパッケージはreview-style.styですでに読み込まれているのでusepackageは意味がないです(エラーにもなりませんが)。
